benches/benchmarks/message.rs

```diff
@@ -59,7 +59,7 @@ fn bench_message(c: &mut Criterion) {
     for size in &sizes {
         g.throughput(Throughput::BytesDecimal(*size as u64));
         g.bench_with_input(
-            BenchmarkId::new("encrypt_password_s2k_iter_aes128", size),
+            BenchmarkId::new("encrypt_password_s2k_iter_aes128_seipdv1", size),
```

Comment:
what about benchmarks for seipdv2?

Reply:
There is #388, for this. I imagined we'd do that after landing this PR (but am open to doing it before, as well).

Reply:
ah right, I forgot about it 😅 can land later